mesonbuild/scripts/meson_exe.py

```diff
@@ -68,7 +73,7 @@ def run(args):
         print(sys.argv[0] + ' [data file]')
     exe_data_file = options.args[0]
     exe = pickle.load(open(exe_data_file, 'rb'))
-    run_exe(exe)
+    return run_exe(exe)
```

Comment:
Note that properly exiting with the wrapped command's exit code means that, in the case of an error, ninja will now print out the command line for this target, which is the wrapped version (python3 meson --internal exe ...). I don't know if that's a problem, but previously, it'd probably just ignore the error entirely.